doc/devel/coding_guide.rst

```diff
@@ -110,8 +110,9 @@ Documentation
   :file:`doc/users/whats_new.rst`.
 
 * If you change the API in a backward-incompatible way, please
-  document it in the relevant file in most recent
-  :file:`doc/api/api_changes_X.Y`.
+  document it by adding a file in the relevant subdirectory of
+  :file:`doc/api/next_api_changes/`, probably in the ``behaviour/``
```

Comment:
```suggestion
  :file:`doc/api/next_api_changes/`, probably in the ``behavior/``
```

Reply:
It *was* named behaviour, then I realized I'd get in trouble for that ;-)

Reply:
Well, someone did get away with it once... Just have to be consistently subtle with it ;)